src/state/slices/assetsSlice/selectors.ts

```diff
@@ -58,7 +58,7 @@ export const selectAssetsByMarketCap = createSelector(
 const chainIdFeeAssetReferenceMap = (chain: ChainTypes, network: NetworkTypes): AssetReference => {
   if (chain === ChainTypes.Bitcoin) return ASSET_REFERENCE.Bitcoin
   if (chain === ChainTypes.Ethereum) return ASSET_REFERENCE.Ethereum
-  if (chain === ChainTypes.Cosmos) {
+  if (chain === ChainTypes.Cosmos || chain === ChainTypes.Osmosis) {
```

Comment:
it would be nice to be able to use the utils from the caip package to do this, rather than the view layer doing caipy things. 

if this doesn't need to go in urgently - please use https://github.com/shapeshift/lib/pull/676 when its' ready